src/test/java/pl/touk/parking/TimeUtil.java

```diff
@@ -0,0 +1,17 @@
+package pl.touk.parking;
+
+import java.time.*;
+
+public class TimeUtil {
```

Comment:
jak ci się kojarzy słówko `util`?

Reply:
Z klasami zawierającymi ogólne metody, które są wykorzystywane w wielu miejscach, a nie bardzo pasują nigdzie indziej?

Reply:
Mi z ogniem piekielnym xD Polecam mój jedyny blog post: https://mateuszrasinski.github.io/do-not-create-public-static-methods/

Aczkolwiek w testach są jeszcze OK, ale wtedy nazwałbym bardziej `~Fixture`.